packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -35,14 +35,17 @@ const Autocomplete = React.memo(props => {
         dropdown_offset,
         error,
         has_updating_list = true,
+        hide_list = false,
         input_id,
         is_alignment_top,
         list_items,
         list_portal_id,
         onHideDropdownList,
         onItemSelection,
         onScrollStop,
+        onShowDropdownList,
         should_filter_by_char,
+        show_list = false,
```

Comment:
show list is used to automatically show the dropdown list for editing the payment method, when autocomplete is rendered